images/router/haproxy/conf/haproxy-config.template

```diff
@@ -60,34 +60,15 @@ global
   # Prevent vulnerability to POODLE attacks
   ssl-default-bind-options no-sslv3
 
-# The default cipher suite can be selected from the three sets recommended by https://wiki.mozilla.org/Security/Server_Side_TLS,
-# or the user can provide one using the ROUTER_CIPHERS environment variable.
-# By default when a cipher set is not provided, intermediate is used.
-{{- if eq (env "ROUTER_CIPHERS" "intermediate") "modern" }}
-  # Modern cipher suite (no legacy browser support) from https://wiki.mozilla.org/Security/Server_Side_TLS
-  tune.ssl.default-dh-param 2048
-  ssl-default-bind-ciphers ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256
-{{ else }}
-
-  {{- if eq (env "ROUTER_CIPHERS" "intermediate") "intermediate" }}
-  # Intermediate cipher suite (default) from https://wiki.mozilla.org/Security/Server_Side_TLS
+# Cipher suite is provided by the ROUTER_CIPHERS environment variable.
+# The default is `intermediate` from the recommendations at https://wiki.mozilla.org/Security/Server_Side_TLS.
+{{- if ne (env "ROUTER_CIPHERS") "" }}
+  tune.ssl.default-dh-param {{env "DH_PARAM_SIZE" "2048"}}
```

Comment:
Not even sure we need the 2048 default here since we want both the cipher set and the param size to be required